src/node_sockaddr-inl.h

```diff
@@ -61,7 +61,6 @@ bool SocketAddress::Compare::operator()(
 bool SocketAddress::Compare::operator()(
     const sockaddr* laddr,
     const sockaddr* raddr) const {
-  CHECK(laddr->sa_family == AF_INET || laddr->sa_family == AF_INET6);
```

Comment:
Uh… `GetLength()` only handles IPv4 or IPv6 addresses, so removing this doesn’t seem right?

Reply:
Ah, I meant to comment this out to temporarily work around a problem. There's a case where this check fails and I haven't narrowed it down yet